src/Bicep.Core.Samples/Files/DisableNextLineDiagnosticsDirective_CRLF/main.sourcemap.json

```diff
@@ -0,0 +1,134 @@
+{
+  "12": {
+    "Item1": "main.bicep",
+    "Item2": 18
+  },
+  "13": {
```

Comment:
Thinking about how you're representing relative paths in this file - you may need to indicate which file is the 'entry point' for the compilation.

As an example, if you have the following 3 files, it will be ambiguous when interpreting the source map:
* main.bicep
* ../main.bicep
* path/to/main.bicep

Reply:
Glad you caught this, I forgot to revisit this issue. I got lost in thinking about edge cases for this if I were to make all file paths relative to the "entry point" source file. Things like modules on separate drives and such. Let me know if you have any thoughts around this.

Reply:
Good point, will have to think of something.